enterprise/frontend/src/embedding-sdk/components/public/dashboard/EditableDashboard/EditableDashboard.stories.tsx

```diff
@@ -135,16 +151,23 @@ export default {
       action: "onLoadWithoutCards",
     },
   },
-};
+  render: ({ "queryBuilderProps.entityTypes": entityTypes, ...args }) => {
```

Comment:
Nice trick, at first I thought Storybook has a custom feature that turns dot notations into controls 😆

Reply:
No, it doesn't :) I found [an addon](https://github.com/eliasm307/storybook-addon-deep-controls) that will do that, but I think it's overkill for a single option, so I just do it manually for now.